### PR TITLE
Added TobezVia is-a-dev-cli

### PR DESCRIPTION
**Pull request by tobezdev**
Added Tobezvia the CLI

### CODE REVIEW

File: domains/Tobez.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (WilliamDavidHarrison) [inline domains/Tobez.json:1]: Delete this file please.

File: domains/tobez.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "TheRealTobes28-Codez",
+        "email": "null"
+      },
+      "record": {
+        "CNAME": "https://tobestech.com/"
```
Comment (WilliamDavidHarrison) [inline domains/tobez.json:8]:
```suggestion
        "URL": "https://tobestech.com/"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by WilliamDavidHarrison** [comment]
@TheRealTobes28-Codez I need you to delete the file with the capital letter.

File: domains/Tobez.json (hunk 1 above)
Comment (TheRealTobes28-Codez) [reply domains/Tobez.json:1]: Not gonna lie, can’t find a way to do it on my phone :/

Comment (WilliamDavidHarrison) [reply domains/Tobez.json:1]: @TheRealTobes28-Codez https://github.com/TheRealTobes28-Codez/register/blob/main/domains/Tobez.json click this link and click the delete button

**Comment by TheRealTobes28-Codez** [comment]
I would, if there was one ://

**Review by MaskDuck — DISMISSED** [review]

File: domains/tobez.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "TheRealTobes28-Codez",
+        "email": "null"
```
[the remark this answers is not in the document]
Comment (TheRealTobes28-Codez) [reply domains/tobez.json:5]: No idea why it isn't there but it would be therealtobes28@gmail.com

Comment (WilliamDavidHarrison) [inline domains/tobez.json:5]:
```suggestion
        "email": "therealtobes28@gmail.com"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/tobez.json (hunk 3 above)
Comment (WilliamDavidHarrison) [inline domains/tobez.json:5]:
```suggestion
        "email": "therealtobes28@gmail.com"
```

File: domains/tobez.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+      },
+      "record": {
+        "URL": "https://tobestech.com/"
```
Comment (WilliamDavidHarrison) [inline domains/tobez.json:8]: 404

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
404

**Comment by WilliamDavidHarrison** [comment]
Too long without response. Still a 404.

**Pull request closed without merging**